core/pkg/ingress/controller/controller.go

```diff
@@ -138,8 +138,9 @@ type Configuration struct {
 	// (for instance NGINX)
 	Backend ingress.Controller
 
-	UpdateStatus bool
-	ElectionID   string
+	UpdateStatus           bool
+	ElectionID             string
+	StatusUpdateOnShutdown bool
```

Comment:
Another nit - I'm not sure if this is exposed anywhere, but it probably should be `UpdateStatusOnShutdown` to mirror the flag name